lib/templates/lab5/post.dart

```diff
@@ -1,15 +1,35 @@
-// TODO add dependencies
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+import 'package:json_annotation/json_annotation.dart';
 
+part 'post.g.dart';
+
+@JsonSerializable()
```

Comment:
in task with posts, I should by your self implement fabric without other packages